modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -338,8 +346,23 @@ private void configureExternalPorts() {
     private void configureIndexer() {
         logger().debug("Configuring the indexer service");
 
+        String storageMode;
+        switch (indexStorageMode) {
+            case MEMORY_OPTIMIZED:
+                storageMode = "memory_optimized";
```

Comment:
would it make sense to move this constant into the enum itself and remove the need for a switch statement?

Reply:
I'm going to clean that up, rebase and get it freshned up.